autorest/codegen/models/primitive_schemas.py

```diff
@@ -220,9 +220,13 @@ def get_serialization_type(self):
         return "bytearray"
 
     def get_python_type(self, namespace=None):
+        if self.format == ByteArraySchema.Formats.base64url:
+            return "bytes"
         return "bytearray"
 
     def get_declaration(self, value) -> str:
+        if self.format == ByteArraySchema.Formats.base64url:
+            return f'bytes("{value}", encoding="utf-8")'
```

Comment:
That's likely not correct, but we don't have base64 constant in any Swagger right now, so guess we'll see if that happen
https://github.com/Azure/autorest.testserver/issues/124